src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/UrlRewriteParseException.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Microsoft.AspNetCore.Rewrite.Internal.IISUrlRewrite
+{
+    public class UrlRewriteParseException : FormatException
```

Comment:
I think we should change the name of this exception. Maybe InvalidUrlRewriteFormatException. Just spit balling